backend/internal/repository/get_image.go

```diff
@@ -1,36 +1,30 @@
 package repository
 
 import (
+	"fmt"
 	"log"
 
-	"math/rand"
-	"time"
-
 	"github.com/hoka-isdl/MERS/backend/internal/schema"
 )
 
-func GetImages() []schema.ListImagesInner {
+func RandGetImages(images_num int) ([]schema.ListImagesInner, error) {
```

Comment:
引数の変数名はこんな感じにしようか。Goではスネークケースではなくてキャメルケースで統一することを覚えておいてほしい！
```suggestion
func RandGetImages(numImages int) ([]schema.ListImagesInner, error) {
```

Reply:
キャメルケースで統一することを失念しておりました．
変更いたしましたのでご確認お願いいたします．

commit url：[ea735f7](https://github.com/hoka-isdl/MERS/pull/19/commits/ea735f77d9a00ba08291610491375bdd603fcac0)

Reply:
commit urlもあるの助かる！